src/metaBlob.cxx

```diff
@@ -134,12 +134,18 @@ PointDim(void) const
 }
 
 void MetaBlob::
-NPoints(size_t npnt)
+NPoints(int npnt)
 {
   m_NPoints = npnt;
 }
 
-size_t MetaBlob::
+void MetaBlob::
+NPoints(size_t npnt)
```

Comment:
Indeed, if it is "int" internally, a static_cast to a larger type at the interface is not going to have any benefit.